### PR TITLE
feat: add navigateTo resources and edit container connection

### PR DESCRIPTION
**Pull request by lstocchi**
### What does this PR do?

This PR allows to navigate to the resources page and the edit container connection page.
It will be used by AI Lab to help the user when its env is not properly set.
E.g.
You don't have a podman machine running -> go to resources
Your podman machine has not enough resources -> edit it

### Screenshot / video of UI

N/A

### What issues does this PR fix or reference?

N/A

### How to test this PR?

run tests

- [x] Tests are covering the bug fix or the new feature

### CODE REVIEW

File: packages/extension-api/src/extension-api.d.ts [hunk 1]
```diff
@@ -4175,6 +4175,15 @@ declare module '@podman-desktop/api' {
...
      */
     export function navigateToAuthentication(): Promise<void>;
+
+    /**
+     * Navigate to Preferences page
+     */
+    export function navigateToResources(): Promise<void>;
+    /**
+     * Navigate to Preferences page
+     */
```
Comment (benoitf) [inline packages/extension-api/src/extension-api.d.ts]: looks invalid there ?

File: packages/extension-api/src/extension-api.d.ts [hunk 2]
```diff
@@ -4175,6 +4175,15 @@ declare module '@podman-desktop/api' {
      * Navigate to Authentication settings page
      */
     export function navigateToAuthentication(): Promise<void>;
+
+    /**
+     * Navigate to Preferences page
```
Comment (benoitf) [inline packages/extension-api/src/extension-api.d.ts]: looks invalid there ?

**Review by benoitf — APPROVED** [review]
looks like invalid typedoc else LGTM

File: packages/extension-api/src/extension-api.d.ts (hunk 1 above)
Comment (lstocchi) [reply packages/extension-api/src/extension-api.d.ts]: fixed

File: packages/extension-api/src/extension-api.d.ts (hunk 2 above)
Comment (lstocchi) [reply packages/extension-api/src/extension-api.d.ts]: fixed

**Review by axel7083 — APPROVED** [review]
LGTM codewise!

**Pull request merged**